# Logfile created on 2020-02-20 04:53:12 +0000 by logger.rb/66358
{"severity":"ERROR","time":"2020-02-20T04:53:12.346Z","correlation_id":null,"extra.server":{"os":{"name":"Linux","version":"#86~16.04.1-Ubuntu SMP Mon Jan 20 11:02:50 UTC 2020","build":"4.15.0-76-generic","kernel_version":"Linux 1025330cbd81 4.15.0-76-generic #86~16.04.1-Ubuntu SMP Mon Jan 20 11:02:50 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux"},"runtime":{"name":"ruby","version":"ruby 2.6.5p114 (2019-10-01 revision 67812) [x86_64-linux]"}},"exception.class":"Gitlab::Runtime::UnknownProcessError","exception.message":"Failed to identify runtime for process 1088 (bin/rails)","exception.backtrace":["lib/gitlab/runtime.rb:24:in `identify'","config/environment.rb:6:in `\u003ctop (required)\u003e'"]}
